pkg/controller/volume/attachdetach/statusupdater/node_status_updater.go

```diff
@@ -98,7 +98,6 @@ func (nsu *nodeStatusUpdater) processNodeVolumes(nodeName types.NodeName, attach
 		// For all other errors, log error and reset flag statusUpdateNeeded
```

Comment:
The current logic leaves the note with update not needed on the IsNotFound errors, I think we want to maintain that logic, to avoid retries on a node that does not exist?